lib/src/cli/metrics/metrics_command.dart

```diff
@@ -165,50 +165,49 @@ extension _StringEx on String {
   Digest strongHash({List<int> salt = _defaultSalt}) => sha256.convert([...salt, ...utf8.encode(this)]);
 }
 
+Future<FlutterInfo?> getInfo(Options options) async {
```

Comment:
Weird diff

Reply:
I deleted the old command code and probably that's why its confused